scripts/simplify_network.py

```diff
@@ -658,7 +659,11 @@ def merge_isolated_nodes(n, threshold, aggregation_strategies=dict()):
 
     # all the nodes to be merged should be mapped into a single node
     map_isolated_node_by_country = (
-        n.buses.loc[i_suffic_load].groupby("country")["bus_id"].first().to_dict()
+        n.buses.assign(bus_id=n.buses.index.get_level_values(0))
```

Comment:
Sorry a comment: why get_level_values here? isn't buses.index enough?

Reply:
Our data structure seem to be quite flexible, while multi-indexing is a usual pythonic practice. So, I thought it could be natural to use a general approach

However, looking into the code, it seems that we assume everywhere that n.buses always only one index. So, `get_level_values(0)` looks like over-complication. Removed